lib/bokken/ecto/pt_mobile.ex

```diff
@@ -1,7 +1,27 @@
 defmodule Bokken.Ecto.PtMobile do
-  @moduledoc """
+  @moduledoc false
+
+  @doc ~S"""
   PT Phone number type with validation and formatting for Ecto.
+
+  ## Examples
+
+    iex> Bokken.Ecto.PtMobile.cast("+351 912 345 678")
+    {:ok, "+351912345678"}
+    
+    iex> Bokken.Ecto.PtMobile.cast("929066855")
+    {:ok, "+351929066855"}
+
+    iex> Bokken.Ecto.PtMobile.cast("+351 939-066-855")
+    {:ok, "+351939066855"}
+
+    iex> Bokken.Ecto.PtMobile.cast("+351 979 066 855")
+    {:error, [message: "invalid PT phone number"]}
+
+    iex> Bokken.Ecto.PtMobile.cast("+351 989 066 855")
+    {:error, [message: "invalid PT phone number"]}
   """
+  import BokkenWeb.Gettext
 
   use Ecto.Type
```

Comment:
You should keep use first, tacking into account it's special nature.

3rd point on CeSIUM's guides for Elixir: https://github.com/cesium/guides/tree/main/elixir#elixir

@danielsp45 @RuiL1904 @ruioliveira02 (I think this is the purpose of a style guide).